R/transformCounts.R

```diff
@@ -207,6 +206,16 @@ setMethod("transformCounts", signature = c(x = "SummarizedExperiment"),
                  "different from `abund_values`.",
                  call. = FALSE)
         }
+        # Check method
+        # If method is not single string, user has not specified transform method,
+        # or has given e.g. a vector
+        if(!.is_non_empty_string(method)){
+          stop("'method' must be a non-empty single character value. \n",
+               "Give one method from the following list: \n",
+               "'clr', 'hellinger', 'log10', 'pa', 'rank', 'relabundance'",
```

Comment:
This should be removed, because that feature is not tested for in this `if`. I know, this might be helpful for a user at first glance, but essentially it is a crutch.

The user should always to a single point of truth and that is the man page, not an error message.

Reply:
That's true, methods are not checked in this point. It is just checked that method is not empty

Changed